### PR TITLE
"create ingredient in the list, delete an ingredient from the list delete the list, show the list & edit schema to add units to the list_ingredient"

### PR DESCRIPTION
**Pull request by syboulette**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -133,13 +134,13 @@
   end
 
   create_table "recipes", force: :cascade do |t|
-    t.integer "prep_time"
+    t.time "prep_time"
```
Comment (Za-hell) [inline db/schema.rb:137]: I think Ariane has changed the data type from **time** to **integer** - if this change goes trough her code gonna break.

**Review by Za-hell — APPROVED** [review]

File: app/controllers/list_ingredients_controller.rb [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    authorize @list_ingredient
+    @list_ingredient.destroy
+    redirect_to list_path(@list_ingredient.list), status: :see_other, notice: "The ingredient has been deleted!"
+    end
```
Comment (D-Zaleski) [inline app/controllers/list_ingredients_controller.rb:35]: indentation is off for the "end"

File: app/controllers/lists_controller.rb [hunk 3]
```diff
@@ -9,24 +9,22 @@ def index
...
     @list.user = current_user
-    @fridge_ingredients = FridgeIngredient.find(params[:fridge_ingredient_id])
-    @list.fridge_ingredient = @fridge_ingredient
 
     authorize @list
 
-    if @list.save!
-      redirect_to list_path(@list)
+    if @list.save
+      redirect_to lists_path
```
Comment (D-Zaleski) [inline app/controllers/lists_controller.rb:27]: are you trying to redirect to the list index? This goes for all other instances where you've used "listS_path"

File: app/policies/list_policy.rb [hunk 4]
```diff
@@ -4,28 +4,29 @@ def resolve
...
+    true
   end
 
   def edit?
     update?
   end
 
   def update?
-    record.user == user
+    true
```
Comment (D-Zaleski) [inline app/policies/list_policy.rb:25]: record.user == user is correct.

File: app/policies/list_policy.rb [hunk 5]
```diff
@@ -4,28 +4,29 @@ def resolve
...
   end
 
   def destroy?
-    record.user == user
+    true
```
Comment (D-Zaleski) [inline app/policies/list_policy.rb:29]: same thing as above

File: app/controllers/lists_controller.rb (hunk 3 above)
Comment (syboulette) [reply app/controllers/lists_controller.rb:27]: Hi Dom, this was done and checked by Vlad so all good :)

**Pull request merged**